src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextUI.java

```diff
@@ -1029,14 +1029,17 @@ public Dimension getMaximumSize(JComponent c) {
 
 
     /**
-     * Gets the allocation to give the root View.  Due
-     * to an unfortunate set of historical events this
-     * method is inappropriately named.  The Rectangle
-     * returned has nothing to do with visibility.
+     * Gets the allocation for the root view.
```

Comment:
What I've been asking for is something like
"Gets the allocation (i.e. the allocated size) for the root view."

A CSR doesn't seem strictly necessary but I recommend one anyway